bot/internal/bot/label_test.go

```diff
@@ -56,7 +56,7 @@ func TestLabel(t *testing.T) {
 					Deletions: 10,
 				},
 			},
-			labels: []string{"documentation", string(small)},
+			labels: []string{"documentation", "no-changelog", string(small)},
```

Comment:
Same as above (const vs magic string)

Reply:
Fixed in 9078af911d64ccf6459280851bf0fb783acc7258